enforcer/utils/convert_mysql_to_sqlite

```diff
@@ -42,6 +42,6 @@ sqlite3 $DB_OUT < $SCHEMA
 
 echo "Converting database"
 
-mysqldump -u $DB_USR -p$DB_PWD -h $DB_HOST $DB_IN | awk '/INSERT INTO/,/;/' | sed -r 's/^INSERT INTO `([^"]+)`/INSERT INTO "\1"/' > tmp
+mysqldump --no-create-db --no-create-info --user="$DB_USR" --password="$DB_PWD" --host="$DB_HOST" "$DB_IN" | sed -r 's/^INSERT INTO `([^"]+)`/INSERT INTO "\1"/' > tmp
```

Comment:
The awk was used to capture INSERT statements that span multiple lines due to newlines in strings. Please put it back or provide alternative.